Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+Serialization.swift

```diff
@@ -174,7 +174,7 @@ extension PathHierarchyBasedLinkResolver {
         }
         
         return SerializableLinkResolutionInformation(
-            version: .init(major: 0, minor: 0, patch: 1), // This is still in development
+            version: .init(major: 0, minor: 1, patch: 0), // This is still in development
```

Comment:
How this is related to the crash?

Reply:
This is a consequence of changing the modules from a dictionary to an array which also changed the file format of the serialized path hierarchy. Since that file format change isn't backwards compatible I bumped the version number.